app/ui/src/app/integration/edit-page/action-configure/action-configure.component.html

```diff
@@ -28,84 +28,88 @@
       </ng-container>
     </button>
   </ng-template>
-  <div class="main-panel-inner action-configure">
+  <div class="action-configure syn-scrollable">
```

Comment:
I wonder if we really need to keep the page-specific classes here?

Reply:
They're no longer used, removed.